### ISSUE TITLE
Add a whiteBoard 

### ISSUE THREAD

**Issue by LeGmask** [schoolsyst/webapp#100]
When you brainstorm whiteBoard is a really interesting thing to wrote idea with link !

**Comment by ewen-lbh** [comment]
I don't really understand what you mean. Can you rephrase it?

**Comment by LeGmask** [comment]
A whiteBoard could be really cool, because when I'm working i like write idea with arrow and schema ...

**Comment by ewen-lbh** [comment]
Ooh I see 


Basically a place to draw stuff


'kay, will add to the way-too-long list of features to implement :)
